src/components/unneededitems.tsx

```diff
@@ -154,7 +182,7 @@ class UnneededItems extends Component<UnneededItemsProps, UnneededItemsState> {
 										maxRarity={item.rarity}
 										rarity={item.rarity}
 									/>
-									<p>{item.name}</p>
+									<p><a href={`https://sttwiki.org/wiki/${item.name.replace(/\s/g,'_')}`}>{item.name}</a></p>
```

Comment:
I think you might need to encodeURIComponent the name here. Some of the generated links didn't work when I clicked them. For example, I think apostrophes become `%27` on the wiki.
https://sttwiki.org/wiki/Archer%27s_Ritual_Pants

Reply:
Good catch, thanks for pointing that out!